SPECS/glusterfs/glusterfs.spec

```
@@ -1216,16 +1438,23 @@ exit 0
 %dir %{_sharedstatedir}/glusterd
 %dir %{_sharedstatedir}/glusterd/events
 %dir %{_libexecdir}/glusterfs
-%dir %{_libexecdir}/glusterfs/gfevents
-%{_libexecdir}/glusterfs/gfevents/*
-%{_libexecdir}/glusterfs/peer_eventsapi.py*
+     %{_libexecdir}/glusterfs/gfevents
+     %{_libexecdir}/glusterfs/peer_eventsapi.py*
 %{_sbindir}/glustereventsd
 %{_sbindir}/gluster-eventsapi
 %{_datadir}/glusterfs/scripts/eventsdash.py*
+%if ( 0%{?_with_systemd:1} )
 %{_unitdir}/glustereventsd.service
+%else
+%{_sysconfdir}/init.d/glustereventsd
+%endif
 %endif
 
 %changelog
+* Thu Dec 14 2023  Brian Fjeldstad <bfjelds@microsoft.com> - 11.1-1
+- Upgrade to 11.1
```

In case of updates where we use other distro's specs to guide us, please also include a small note about the version to both help future us to understand where the upgrade came from and to give credit to the other authors. For example (taken from other specs):
```suggestion
- Upgrade to 11.1 using Fedora 39 spec for guidance (license: MIT).
```